src/types/venues.ts

```diff
@@ -378,6 +382,8 @@ export interface VenueEvent {
   id?: string;
```

Comment:
[OOS]
Just a reminder the branded strings aren't useful as one or two lines. 
IDs might be the ones that pose more danger of mixup than slugs.

```suggestion
  id?: SpaceId;
```